docs/kb/pcr-measurements.md

```diff
@@ -0,0 +1,157 @@
+|          |                                  |                 | 61dfe48b-ca93-d211-aa0d-00e098032b8c:Boot0000... variables
+| PCR-4    | EV_EFI_ACTION                    | Any boot try    | "Calling EFI Application from Boot Option"
+| PCR-0..7 | EV_SEPARATOR                     | Always          | Separator
+| PCR-1    | EV_EFI_HANDOFF_TABLES            | First boot try  | SMBIOS base+size (could use v2 of the event, but doesn't ATM)
+| PCR-4    | EV_EFI_BOOT_SERVICES_APPLICATION | Any boot try    | Hash of an EFI application (bootloader, UEFI shell, etc.)
+| PCR-7    | EV_EFI_VARIABLE_AUTHORITY        | Unknown         | Measurements done by a bootloader (GRUB, shim, etc.), listed here to show where they appear
+| PCR-8..9 | EV_IPL                           | Unknown         | Measurements done by a bootloader (GRUB, shim, etc.), listed here to show where they appear
+| PCR-5    | EV_EFI_ACTION                    | OS is booting   | "Exit Boot Services Invocation"
+|          |                                  |                 | "Exit Boot Services Returned with Success"
+
+## Other sources of measurements
+
+Applications started by firmware to boot the system continue to measure code
+and data they use and also show up in the event log.  You can find more details
+in the documentation of respective tools (e.g., for [GRUB2][grub] or
+[shim][shim]).
+
+[grub]: https://www.gnu.org/software/grub/manual/grub/html_node/Measured-Boot.html
+[shim]: https://github.com/rhboot/shim/blob/master/README.tpm
+
+## Caveats
+
+### Accuracy
+
+Exact list of measurements depends on the combination of firmware components,
+their versions, installed hardware, TPM version, runtime settings and at least
+user actions.  This is why this document can only enumerate things which
+typically appear in event logs and can't go much further.
+
+#### Repeated entries
+
+Sometimes the same thing is measured multiple times (usually to the same PCR,
+digest is the same if that thing is unchanged) or many similar things get
+measured in sequence.  Keep this in mind when looking for something in the event
+log as it can sometimes appear more than once, including several times in a row.
+
+#### Log entries changing values
+
+Bootloaders or components of operating systems might update SecureBoot
+configuration automatically thus changing PCR values.  Similarly, any kind of
+software update or even update of configuration can potentially change
+behavior and cause an unexpected PCR value.  In such cases PCR values tend to
+stability after a reboot.
+
+#### Conditional measurements
+
+Some things are measured dynamically when they get used which in turn is
+controlled by some setting.  This can result in a single configuration change
+affecting event log in multiple places, for example: disabling a device will
+change value of a corresponding option and possibly remove an entry with a
+measurement of its OptionROM.
```

Comment:
I would also briefly mention user actions (setup, boot menu) here. It seems to be one of the most common causes of unexpected values, at least in my experience.